conda-envs/mlos-windows.yml

```diff
@@ -16,7 +16,8 @@ dependencies:
   - jupyter
   - ipykernel
   - nb_conda_kernels
-  - matplotlib
+  - matplotlib<3.9
+  - matplotlib-base<3.9
```

Comment:
@motus do we need `pyqt` at all?
https://conda-forge.org/docs/maintainer/knowledge_base/#matplotlib

Reply:
good question! Maybe we can get by with just `matplotlib-base` here and in `setup.py`? The reason I've added `matplotlib-base` explicitly was because without it our environment ends up with `matplotlib=3.9.1` and `matplotlib-base=3.8.4` both installed, and that led to conflicts in (i believe) common binaries

Reply:
anyway, let me experiment with it now. stay tuned! :)

Reply:
`matplotlib-base` would only be in `conda` - it's not a `pypi` package

Reply:
yep, figured that out already. everything seems to work with `matplotlib-base` only